mlir/lib/IR/AffineExpr.cpp

```diff
@@ -1326,14 +1334,13 @@ void SimpleAffineExprFlattener::visitDivExpr(AffineBinaryOpExpr expr,
                                              localExprs, context);
```

Comment:
Can we also go into this branch if `getRHS` is a constant expr that is `0`?

Reply:
run [issue](https://github.com/llvm/llvm-project/issues/64622) test case, will going into this branch.